datasafe-long-run-tests/datasafe-business-tests-random-actions/src/test/java/de/adorsys/datasafe/business/impl/e2e/randomactions/framework/BaseRandomActions.java

```diff
@@ -78,12 +79,57 @@ protected Fixture fixture(String path) {
     @ValueSource
     protected static Stream<Arguments> actionsOnSoragesAndThreadsAndFilesizes() {
         return Sets.cartesianProduct(
-                Collections.singleton(minio()),
+                Collections.singleton(s3()),
             THREAD_COUNT,
             FILE_SIZE_M_BYTES
         ).stream().map(it -> Arguments.of(it.get(0), it.get(1), it.get(2)));
     }
 
+    @ValueSource
+    protected static Stream<Arguments> actionsOnMultiStorageParallelThreads() {
+        List<String> users = new ArrayList<>();
+        smallFixture().getOperations().stream().forEach((i)->users.add(i.getUserId()));
```

Comment:
Can be 
`smallFixture().getOperations().forEach((it->users.add(it.getUserId()));`
Typically `it` not `i` is name of lambda parameter that is iterated on

Reply:
Updated

Reply:
updated